### PR TITLE
Update dashboard

### PR DESCRIPTION
**Pull request by keks**
This updates both things that we already fixed, and things fixed in https://github.com/openmls/openmls/pull/1684

### CODE REVIEW

File: checksets/01-leaf_node.dhall [hunk 1]
```diff
@@ -27,7 +27,7 @@ let checks =
             types.CodeRefs/empty
             types.CodeRefs/empty
             ( types.Notes/single
-                "maybe this should be an application-level check?"
+                "This is about what leaf nodes get passed into OpenMLS rather than what we do with them. Therefore this is more of an application-level check."
```
Comment (franziskuskiefer) [inline checksets/01-leaf_node.dhall:30]: Let's be definitive on what we consider this and drop the "more of".

File: checksets/01-leaf_node.dhall [hunk 2]
```diff
@@ -121,10 +121,10 @@ let checks =
                 ''
                 [ "section-7.3-4.5.1", "section-7.3-4.5.2.2" ]
             )
-            types.Status.Missing
+            types.Status.Partial
             types.CodeRefs/empty
             types.CodeRefs/empty
-            types.Notes/empty
+            (types.Notes/single "check is implemented, still need to test it.")
```
Comment (franziskuskiefer) [inline checksets/01-leaf_node.dhall:127]:
```suggestion
            (types.Notes/single "check is implemented, still needs to be tested.")
```

**Review by franziskuskiefer — APPROVED** [review]
lgtm with a few nits

**Comment by keks** [comment]
Waiting for https://github.com/openmls/openmls/pull/1684 to be merged before merging this one

**Pull request merged**